SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentReader.cpp

```diff
@@ -567,7 +567,8 @@ void ComponentReader::ApplyProperty(Schema_Object* Object, Schema_FieldId FieldI
 	}
 	else
 	{
-		checkf(false, TEXT("Tried to read unknown property in field %d"), FieldId);
+		UE_LOG(LogSpatialComponentReader, Error, TEXT("Tried to read unknown property in field %d. Property name %s."), FieldId,
```

Comment:
```suggestion
		UE_LOG(LogSpatialComponentReader, Error, TEXT("Tried to set unknown property or unsupported property type while applying schema component field %d. Property name: %s."), FieldId,
```

Reply:
I don't think it's easily available in this function, but I wish we could log out the component ID in addition to the field ID here :/

Reply:
Oh also, if we happen to know that our `Property` pointer here is always valid, we should only hit this when the property type is unsupported, in which case we could check the property type and log out a message saying specifically that, which I think would be more helpful to the user. Can be a follow-up ticket if that gets too complicated though.

Reply:
`Tried to set unknown property or read` missed out the read there

Reply:
Ah no that was intentional - I'm not sure why the original check message was "read unknown property", when this function only sets properties.

Reply:
The original thought for this error message was that we failed to read the schema object field, rather than set the Unreal property.

Reply:
`Property` is not fully guaranteed to be non-null, I think, and also getting the type seemed a bit difficult at first glance.